tests/unit/amo/components/TestCategory.js

```diff
@@ -166,17 +340,175 @@ describe('Category', () => {
 
   it('sets loading to true if categories are loading', () => {
     _categoriesFetch();
+
     const root = render({}, { autoDispatchCategories: false });
+    expect(root.instance().props.loading).toEqual(true);
```

Comment:
😅

Reply:
ah nope, that does not work actually because `root` refers to the rendered component, that is the `<div className="Category">` (because we `shallowUntilTarget()` I guess).